Reports/progress_report.md

```diff
@@ -0,0 +1,295 @@
+
+#### Goal
+Ensure that institutions evaluate themselves against SATRE and that the momentum is maintained between funded phases
+
+At the end of the funded phase of SATRE there was a growing community interest, with institutions and stakeholders affirming they would evaluate themselves against it and contribute feedback.
+
+Without ongoing resources the necessary support to ensure that happens cannot be provided and SATRE may end up not being adopted.
+
+#### Definition of Done
+There are a number of self evlauaitons completed, there is feedback on the spec repository and, ideally, there is an active WG within the TRE Community continuing to work on it.
+
+#### Progress
+Work is folded into the [UK TRE Community](https://github.com/alan-turing-institute/data-safe-haven-team/issues/52), already SATRE is formally becoming a UK TRE Community WG.
+
+## Community building
+Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
+Creating and maintaining open and active communication spaces & workspaces (Slack, GH)
+Identifying and documenting everything that can be openly documented
+
+### Stakeholder landscape review
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/30
+
+Contributes to:
+- Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
+- Creating and maintaining open and active communication spaces & workspaces (Slack, GH)
+
+#### Goal
+Across a lot of our work (DSH project, SATRE, UK TRE community) there has been a lot of discussion around who the impacted parties are, how they are categorised, what their interests/needs are etc.
+
+An effective stakeholder map showing all parties we think we should engage with will help us prioritise who to collaborate with, and strengthen our work in community building within the TRE space (which is kind of where this project is heading, above and beyond getting others to use the DSH).
+
+- [ ] Brainstorm and identify potential stakeholder groups
+- [ ] Engage different groups through interviews/workshops to better understand them
+- [ ] Create engagement pipeline & priority for different groups
+
+#### Definition of Done
+- When we have an intended end-output from engagement with our established groups (e.g. by the end of the project, we want X group to be part of the UK TRE community, we want Y group to have contributed to the DSH repo...)
+
+#### Progress
+The work done in [SATRE](https://github.com/alan-turing-institute/data-safe-haven-team/issues/23) and the [UK TRE Community](https://github.com/alan-turing-institute/data-safe-haven-team/issues/52) have directly contributed to estbalishing a relationship with key stakeholders, identify and characterise them.
+
+Direct and explicit work on this story have not been carried out (not scheduled in this period).
+
+We are now establishing the engagement pipleline by creating an internal CRM (sharepoint based currently).
```

Comment:
The CRM is currently structured to be very like informal and just for us to track internally what's going on with different convos. 

If this becomes a significant story, do we want to put effort into formalising this a bit more?

@JimMadge a little bit of an extension of the Slack thread

Reply:
I get the feeling this isn't big enough to be a story. At least not with the level of investment we currently give it.